4-0-object-oriented-programming/4-3-1-flight-search/src/main/java/com/bobocode/oop/service/FlightService.java

```diff
@@ -12,14 +13,16 @@
  */
 public class FlightService {
 
+    FlightDao flightDao = new FlightDao();
```

Comment:
use interface here, and set this dependency with constructor

Reply:
There is no interface in that package, should I create a new one or leave it as it is?
`    FlightDao flightDao;

    public FlightService() {
        flightDao = new FlightDao();
    }
`